test/test_dynamic_transform.cpp

```diff
@@ -618,4 +618,107 @@ TEST_F(NVFuserTest, DynamicTransform11_CUDA) {
       info2.toString());
 }
 
+// Test FusionExecutorCache with dynamic reshapes
+TEST_F(NVFuserTest, DynamicTransformFusionExecutorCache_CUDA) {
+  auto fusion = std::make_unique<Fusion>();
+  FusionGuard fg(fusion.get());
+
+  auto tv0 = makeSymbolicTensor(2);
+  fusion->addInput(tv0);
+  auto tv1 = makeSymbolicTensor(2);
+  fusion->addInput(tv1);
+
+  auto reshape_shape0 = IrBuilder::create<Int>();
+  fusion->addInput(reshape_shape0);
```

Comment:
Is it necessary to register the symbolic int as a fusion input?

Reply:
It's not, since we could use `tv1->axis(0)->extent()` instead as in `DynamicTransform2_CUDA`. I was trying to match `DynamicTransform1_CUDA`, but the `reshape_like` way is probably a little simpler.